package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "voxa-chatbase",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
Since the past PR contained a major update to the framework, it should have been versioned as 1.0.0, now that you are adding this change, I'd suggest to version it as: 1.0.1